yarn.lock

```diff
@@ -16058,7 +16058,7 @@ webpack-virtual-modules@^0.6.1:
   resolved "https://registry.yarnpkg.com/webpack-virtual-modules/-/webpack-virtual-modules-0.6.1.tgz#ac6fdb9c5adb8caecd82ec241c9631b7a3681b6f"
   integrity sha512-poXpCylU7ExuvZK8z+On3kX+S8o/2dQ/SVYueKA0D4WEMXROXgY8Ez50/bQEUmvoSMMrWcrJqCHuhAbsiwg7Dg==
 
-webpack@5, webpack@5.96.1, webpack@^5.64.4:
```

Comment:
let me try rebasing!

Reply:
This was done automatically. Same result after rebasing. I think this is probably just from me running `yarn` for the first time since updating the dependency.